main.js

```diff
@@ -18,10 +20,19 @@ function pressKey(event) {
   $car.className = 'car ' + car.currentDirection;
 }
 
+function startCar() {
+  car.x += 1;
+  $car.style.left = car.x + 'rem';
+  // car.y += 1;
+  // $car.style.top = car.y + 'rem';
```

Comment:
Please remove commented code before submitting

Reply:
Okay will do, thank you!